scripts/eval_turbo_zhko_yue.py

```
@@ -0,0 +1,82 @@
+from eval_accuracy import cer_ja
+from eval_common import load_manifest
+
+ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+MANIFESTS = ["testdata/eval_real_zhko", "testdata/eval_real_yue"]
+
+try:
+    from opencc import OpenCC
+
+    _t2s = OpenCC("t2s").convert
+except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)